test/integration/targets/ansible-doc/randommodule-text.output

```diff
@@ -2,8 +2,6 @@
 
         A random module.
 
-ADDED IN: version 1.0.0 of testns.testcol
```

Comment:
Should this really be removed? I don't think that's a good idea.

Reply:
not removed, it appears on higher verbosity now .. makes it a lot less cluttered in some modules, specially those that added many options after initial version

I might move more things behind verbosity levels, trying to figure out what the good line is